.vscodeignore

```diff
@@ -11,3 +11,6 @@ logs/**
 sessions/**
 gulpfile.js
 docs
+.github
```

Comment:
SHould this be a directory or doesn't matter?

Reply:
Does not appear to matter to vscode, as it kept the directory out. I'll change it anyway